### PR TITLE
Updated roles home

### PR DESCRIPTION
**Pull request by jayp822**
Removed old code and images associated with the HomeCarousel component and updated the roles in the second text block to transition automatically while a viewer is on the home page.

https://github.com/user-attachments/assets/48eb5cc5-f55a-45d0-9533-3e7b46911615

### CODE REVIEW

**Review by justinquinnb — COMMENTED** [review]
Visually looks great to me, I approve. @nivedhanatarajan7 or @KelseySt will ensure adherence to code style and merge.

**Comment by jayp822** [comment]
My recent commit fixes the problem Nivedha brought up about the connecting buttons in the /about directory and the footer not connecting in the our-team page. Here is what it look like now

https://github.com/user-attachments/assets/21bb1b70-3e05-47bd-a468-85b1099d69c0

File: src/app/about/people/page.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+
+   <h3 className="text-black text-[3rem] font-semibold text-center mt-[5rem] p-0"> 2024 - 2025 </h3>
+   <h3 className="text-black text-[3rem] font-bold text-center">Executive Board</h3>
+
+
+
+
+   <div id="grid">
```
Comment (nivedhanatarajan7) [inline src/app/about/people/page.js:30]: The height of the card and the styling of the name and position seems to have changed, potentially from the Tailwind CSS migration. 
image

It should look similar to 
image


Thank you!

**Review by nivedhanatarajan7 — CHANGES_REQUESTED** [review]
Overall, the changes are awesome! I really like the Navbar changes. The styling of the Team cards seem to have changed, so it would be great if you could update this to look similar to how it is in the currently deployed version of the website. Thank you so much!

File: src/app/about/people/page.js (hunk 1 above)
Comment (jayp822) [reply src/app/about/people/page.js:30]: yeah I noticed that too. Should be an easy fix.

**Review by nivedhanatarajan7 — APPROVED** [review]

**Pull request merged**